mnefun/_mnefun.py

```diff
@@ -2330,3 +2332,144 @@ def plot_reconstruction(evoked, origin=(0., 0., 0.04)):
     axs[2, 0].set_ylabel('Reconstruction')
     fig.tight_layout()
     return fig
+
+
+def plot_chpi_snr_raw(raw, win_length, n_harmonics):
+    """Compute and plot cHPI SNR from raw data
+
+    Parameters
+    ----------
+    win_length : float
+        Length of window to use for SNR estimates (seconds). A longer window
+        will naturally include more low frequency power, resulting in lower
+        SNR.
+    n_harmonics : int
```

Comment:
BTW if I understand it correctly, info['lowpass'] is supposed to be the corner frequency (-3 dB?) of the analog anti-aliasing filter, and thus it should always be clearly below Nyquist frequency. With the Elekta system, you cannot set the lowpass corner above `fs/3` (where `fs` is sampling frequency). OTOH, if there is strong interference at frequencies above the lowpass corner, the analog filter may not be able to get rid of it.

I'll see what happens when including freqs upto `info['lowpass'].` or even Nyquist by default. If it works fine, why not.